types/luxon/index.d.ts

```diff
@@ -467,11 +468,11 @@ export interface ZoneOffsetOptions {
 }
 
 export class Zone {
-    static offsetName(ts: number, options?: ZoneOffsetOptions): string;
-    static isValid: boolean;
-    static name: string;
-    static type: string;
-    static universal: boolean;
+    offsetName(ts: number, options?: ZoneOffsetOptions): string;
```

Comment:
From the source it looks like the class is abstract and has abstract methods/getters. You could consider using the `abstract` and `get` syntax here to get even closer to the types from the source.